samples/zigbee/light_bulb/include/zb_dimmable_light.h

```diff
@@ -0,0 +1,207 @@
+/*
+ * ZBOSS Zigbee 3.0
+ *
+ * Copyright (c) 2012-2020 DSR Corporation, Denver CO, USA.
+ * www.dsr-zboss.com
+ * www.dsr-corporation.com
+ * All rights reserved.
+ *
+ *
+ * Use in source and binary forms, redistribution in binary form only, with
```

Comment:
We should use regular Nordic license header here (`* SPDX-License-Identifier: LicenseRef-Nordic-5-Clause`) and in other files

Reply:
Replaced!